backend/src/server.js

```diff
@@ -1,16 +1,20 @@
-const express = require('express');
-const mongoose = require('mongoose');
+const express = require("express");
+const mongoose = require("mongoose");
 
-mongoose.connect('mongodb+srv://mikael:fimika13@mntbk-qboul.mongodb.net/test?retryWrites=true&w=majority', {
```

Comment:
Where will my connection string be added then? I get the basic idea, but I'm not sure about that

Reply:
In the commit #d780f34 I've added the new file called .env.example.

From this file, you will copy to a new one .env (which won't be processed by git).
In the .env file, you should place the connection string and the port where the app will run.